src/app/help/en/faq.md

```diff
@@ -817,6 +818,30 @@ No. None of these actions incur any fees.
 
 A transfer of tokens between accounts typically takes 3 seconds. This is far faster than most blockchain tokens.
 
+<a href="#Table_of_Contents_Economics">^</a>
+## <span id="How_does_bandwidth_work">How does bandwidth work?</span>
+
+Bandwidth represents a share in the resources of the Steem blockchain's network and it is what enables users to perform actions like posting, voting, commenting and making token transfers. Every action a user takes on Steemit, with the exception of viewing content, consumes part of his/her account's bandwidth allocation. All accounts are allocated bandwidth according to how much Steem Power the account holds. The more the Steem Power, the more bandwidth is allocated.
+
+This bandwidth rate-limiting is employed to safeguard the blockchain from spam attacks.
```

Comment:
It may be good to emphasize here that that:
Since transacting on the Steem blockchain has zero fees, bandwidth rate-limiting is employed to safeguard the blockchain from spam attacks.

Reply:
Good point.